packages/office-ui-fabric-react/src/components/FocusTrapZone/FocusTrapZone.test.tsx

```diff
@@ -388,13 +388,13 @@ describe('FocusTrapZone', () => {
       expect(lastFocusedElement).toBe(buttonA);
     });
 
-    it('focuses first focusable element when focusing outside of FTZ with 0 tabbable items', async () => {
+    it('focuses last focusable element when focusing outside of FTZ with 0 tabbable items', async () => {
```

Comment:
Why are we changing this test here?

Reply:
Because I find the UT is not passed before, see https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=257337&view=logs&j=258ec178-2d8b-5611-7b9b-60c5c95dae55.

Looked deeper, find the UT should be updated.